installation/canvas.adoc

```diff
@@ -139,12 +139,14 @@ By default, {CANVAS} is configured to fetch this image from https://quay.io/repo
 
 [IMPORTANT]
 ====
-The pre-configured image is NOT part of BAMOE, and comes from the open-source KIE community. People installing {CANVAS} must change `KIE_SANDBOX_DMN_DEV_DEPLOYMENT_BASE_IMAGE_URL` to make sure the used image meets their requirements. For a reference on how to build your own image that is compatible with {CANVAS} Dev deployments, please go to the KIE community's source code for `dmn-dev-deployment-base-image` at https://github.com/kiegroup/kie-tools/tree/{KIE_TOOLS_COMMUNITY_VERSION}/packages/dmn-dev-deployment-base-image
+The pre-configured image is NOT part of {PRODUCT_SHORT}, and resides in the open-source Apache KIE community.
+If you are  installing {CANVAS}, you must change `KIE_SANDBOX_DMN_DEV_DEPLOYMENT_BASE_IMAGE_URL` to make sure the used image meets your requirements.
+For a reference on how to build your own image that is compatible with {CANVAS} development deployments, please go to the Apache KIE community's source code for `dmn-dev-deployment-base-image` at https://github.com/kiegroup/kie-tools/tree/{KIE_TOOLS_COMMUNITY_VERSION}/packages/dmn-dev-deployment-base-image.
```

Comment:
Shouldn't we use `apache` also here? I mean both links below works while the later one seems to be newer. 
- https://github.com/kiegroup/kie-tools/tree/0.31.0/packages/dmn-dev-deployment-base-image
- https://github.com/apache/incubator-kie-tools/tree/0.31.0/packages/dmn-dev-deployment-base-image

Reply:
Yes, we should, good catch.

Reply:
Fixed